### ISSUE TITLE
Error packaging Ditto chart: can't get a valid version for repositories mongodb

### ISSUE THREAD

**Issue by calohmn** [eclipse/packages#440]
The jenkins website build job currently fails when packaging the Ditto chart:
```
+ /home/jenkins/agent/workspace/Website_master/helm package -u charts/ditto -d helm-repository
Error: can't get a valid version for repositories mongodb. Try changing the version constraint in Chart.yaml
```
There is no matching mongodb chart version (`~11.1.10`) in the current bitnami chart index any more (see also #372).

Oldest mongodb chart version in the `https://charts.bitnami.com/bitnami` index is now `12.1.16`.

**Comment by calohmn** [comment]
Fix is in #439.

**Comment by thjaeckle** [comment]
@calohmn if you need a quick merge of the fix, you could also review/have a look at PR #439 
Otherwise before next week probably noone will have a look.
